notify/notify.go

```diff
@@ -51,14 +56,13 @@ type Kind string
 // Errors.
 var ErrNoRecipient = errors.New("no recipient")
 
-// Init initializes a notifier with the supplied options. See
-// WithSender, WithRecipient, WithFilter, WithStore and WithSecrets
-// for a description of the various options. Secrets are required to
-// send actual emails using the MailJet API, but can be omitted during
-// testing. It is permissable to re-initalize a Notifier with
-// different options, however missing options will revert to their
-// defaults.
-func (n *Notifier) Init(options ...Option) error {
+// NewMailJetNotifier initializes a MailJetNotifier with the supplied
```

Comment:
NewMailJetNotifier -> NewMailjetNotifier

Please just change MailJet to Mailjet everywhere :-)

Reply:
Yes, probably just needed one comment haha :laughing: